test/bitmap-file-helper-test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const expect = require('chai').expect;
+const bitmapFileHelper = require(`${__dirname}/../lib/bitmap-file-helper.js`);
+
+describe('Bitmap File Helper', function() {
```

Comment:
At some point you should test the main point of your program which is the writing new files after the transform.